tests/fixtures/tests.py

```diff
@@ -620,6 +620,40 @@ def test_output_formats(self):
             format='xml', natural_foreign_keys=True
         )
 
+    def test_loading_with_excludes(self):
+        Site.objects.all().delete()
+
+        management.call_command('loaddata', 'fixture1', exclude=['fixtures'], verbosity=0)
+        self.assertFalse(Article.objects.exists())
+        self.assertFalse(Category.objects.exists())
+        self.assertQuerysetEqual(Site.objects.all(), [
+            '<Site: example.com>',
+        ])
+
+        Site.objects.all().delete()
+        Article.objects.all().delete()
```

Comment:
I guess Article/Category deletes aren't doing anything since you already asserted exists() -> False. Anyway, I think I'd put this in a separate method. `test_loading_with_exclude_app` / `test_loading_with_exclude_model`